src/formatDate.js

```diff
@@ -50,7 +50,31 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[fromFormat.length - 1]);
+  const resultArr = [];
+  const tempDate = {};
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    tempDate[fromFormat[i]] = dateArr[i];
+  }
+
+  if (tempDate.hasOwnProperty('YYYY')) {
+    tempDate['YY'] = tempDate['YYYY'].toString().split('').slice(2).join('');
+  }
```

Comment:
no need to do it, if there is no YY on the new format

Reply:
I wrote universal function which collect all possible format and should be able to work with all possible format in result